modules/local/fusioncatcher/build/main.nf

```diff
@@ -2,21 +2,25 @@ process FUSIONCATCHER_BUILD {
     tag "fusioncatcher_build"
     label 'process_medium'
 
-    container "docker.io/rannickscilifelab/fusioncatcher:1.34"
+    conda "${projectDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/d5/d53f36e9e01d14a0ae8e15f8046f52b2883c970c27fe43fdfbd9440a55f5403f/data' :
+        'community.wave.seqera.io/library/fusioncatcher:1.33--4733482b637ef92f' }"
```

Comment:
It works with 1.33 now because ensembl 113 directory does not contain 37 and 38 which was confusing the build. It is not compatible with ensembl 112 and we can't force to use a certain version. Conclusion: use 113/gencode 47 as default

Reply:
And keep 1.33. Add in documentation where to find the references for ensembl 112